Cargo.toml

```diff
@@ -14,7 +14,7 @@ readme = "README.md"
 [features]
 default = [ "secp-recovery" ]
 base64 = [ "base64-compat" ]
-fuzztarget = ["bitcoin_hashes/fuzztarget"]
+fuzztarget = []
```

Comment:
We should drop this feature entirely now, its been replaced with an explicit `--cfg=fuzzing`

Reply:
Is there any chance that people integrated this recusively into their fuzzing pipeline just as we did with `bitcoin_hashes`? If so this is a breaking change.

Reply:
I would be very surprised if that were the case, though, indeed, its possible. Ultimately its still just in fuzzing pipeline(s), so hopefully not anything that would break any release builds anywhere.

Reply:
So I assume we need to leave this as is for `0.26.1` and drop in `0.27`?

Reply:
If we're not going to yank bitcoin_hashes (which I'm fine with) then we can call it a consistent policy. Probably the only thing that will break is rust-Lightning's fuzz crates, and I'm fine dealing with that :).

Reply:
Its a little bit confusing that we now have a `fuzztarget` feature which does nothing at all - you have to set `cfg=fuzzing` to get any behavior out of it. I think we should drop it.